shotover-proxy/tests/transforms/query_type_filter.rs

```diff
@@ -0,0 +1,30 @@
+use crate::helpers::ShotoverManager;
+use serial_test::serial;
+
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_query_type_filter() {
```

Comment:
Can we try pipelining too?

Reply:
Assuming each pipeline message gets represented as a separate message in shotover

Reply:
you were correct!
I added a test case for this very useful case.